### ISSUE TITLE
__init__.py in robotpkg-py27-sot-core-v3

### ISSUE THREAD

**Issue by olivier-stasse** [stack-of-tasks/sot-core#138]
When installing robotpkg-py27-sot-core-v3:
```
more /opt/openrobots/lib/python2.7/site-packages/dynamic_graph/sot/core/__init__.py
```
gives:
```
#
#    Copyright 2010 CNRS
#
#    Author: Florent Lamiraux
#
# This file is free software: you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public License
# as published by the Free Software Foundation, either version 3 of
# the License, or (at your option) any later version.
#
# This file is distributed in the hope that it will be useful, but
# WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
# General Lesser Public License for more details.  You should have
# received a copy of the GNU Lesser General Public License along with
# this file. If not, see <http://www.gnu.org/licenses/>.

import sys
try:
    from DLFCN import RTLD_NOW, RTLD_GLOBAL
except ModuleNotFoundError:  # Python 3
    from os import RTLD_NOW, RTLD_GLOBAL
import dynamic_graph as dg
flags = sys.getdlopenflags()
# Import C++ symbols in a global scope
# This is necessary for signal compiled in different modules to be compatible
sys.setdlopenflags(RTLD_NOW|RTLD_GLOBAL)
from . import wrap
# Recover previous flags
sys.setdlopenflags(flags)

dg.entity.updateEntityClasses(globals())
```
when it should be
```
more ../sot_bionic_ws/install/lib/python2.7/site-packages/dynamic_graph/sot/core/__init__.py
# flake8: noqa
from .exp_moving_avg import *
from .feature_generic import *
from .feature_joint_limits import *
from .feature_point6d import *
from .feature_pose import *
from .feature_position import *
from .feature_position_relative import *
from .feature_posture import *
from .feature_visual_point import *
from .gain_adaptive import *
from .gradient_ascent import *
from .joint_limitator import *
from .kalman import *
from .math_small_entities import *
from .op_point_modifier import *
from .robot_simu import *
from .sot import *
from .task import *
from .task_pd import *
from .visual_point_projecter import *
```
This is what I have when installing from source.
From the binary is does not look like that this repository switch yet to the empty __init__,py policy (to make it relocatable). 
@nim65s what do you want to do to fix that. Right now talos_integration_tests assumes the source base behavior (I am currently making the changes for an empty __init__.py)

**Comment by olivier-stasse** [comment]
Changing this policy is likely to break several python tests & applications.

**Comment by olivier-stasse** [comment]
If all the packages have been updated to reflect the current stage, then only talos_integration_tests needs to be updated.

**Comment by nim65s** [comment]
This file should just not be imported. Where did you find a 
```python
from dynamic_graph.sot.core import …
```

?

**Comment by olivier-stasse** [comment]
https://github.com/stack-of-tasks/talos_integration_tests/blob/651da8398e8fdab7b4116c7c1d46c41926cde9be/scripts/appli_dcmZmpControl_file.py#L6

**Comment by nim65s** [comment]
This import is breaking future work on https://github.com/orgs/stack-of-tasks/projects/2

**Comment by olivier-stasse** [comment]
If this is just a matter of update talos_integration_tests I am fine with it.
But to be consistent the source of sot-core should reflect that.

**Comment by nim65s** [comment]
@olivier-stasse : where are we in this issue ? Did you made any change related to this ? Or should I work on that ?

As a reminder, this issue is currently blocking:
- https://github.com/stack-of-tasks/talos_integration_tests/issues/5
- https://github.com/stack-of-tasks/tsid/pull/87

And therefore the next release of eiquadprog & TSID. Blocking TSID Release also happen to block the binary release of pinocchio 2.4.5.

**Comment by olivier-stasse** [comment]
Fix with commit e88129148
But this will likely breaks some user code even outside robotpkg.
We should warn people on the riot channel.
